_Week8/Group4/README.md

```diff
@@ -0,0 +1,42 @@
+# Week 8 Homework Assignment, Group 4: income_sim
+
+# `simulate.do` Overview
+
+# Our simulate.do file defines a rclass program which takes in sample size as an input.
+# The program firstly creates an index variable `id` to serve as identification, for each variable in the sample size.
+# It creates a `state` variable which ranges from 1 to 10, evenly distributed across observations. For the purpose of the simulation, it is assumed that a higher value for the `state` variable signifies a more urban state.
```

Comment:
If you want to think about making your simulation more complex, you could assign the state variable non-uniformly in order to produce clusters of different sizes